### PR TITLE
Rename encapsulate op to _nGraphEncapsulate

### PR DESCRIPTION
**Pull request by adk9**
* Prefix encapsulation op introduced by our JIT passes with an underscore to _nGraphEncapsulate to
  indicate that it's not a public TF op
* Remove unused function in ngraph_utils.{c,h}

### CODE REVIEW

File: ngraph_bridge/ops/ngraph_ops.cc [hunk 1]
```diff
@@ -1,34 +0,0 @@
-/*******************************************************************************
```
Comment (kanvi-nervana) [inline ngraph_bridge/ops/ngraph_ops.cc:1]: This is a TF convention to keep Op registrations in a separate file from the kernel. Intel-TF also follows the same and I think we should keep this file even though there is only 1 op that we register.

File: ngraph_bridge/ngraph_utils.cc [hunk 2]
```diff
@@ -469,7 +463,7 @@ bool IsProcessedByNgraphPass(Graph* g) {
   // Current method may fail when graph has no encapsulates after first pass
   // Also variable/optimizer change introduces other types of ng nodes
   for (Node* node : g->nodes()) {
-    if (node->type_string() == "NGraphEncapsulate") return true;
+    if (node->type_string() == "_nGraphEncapsulate") return true;
```
Comment (kanvi-nervana) [inline ngraph_bridge/ngraph_utils.cc:466]: Comment at line 464 can be removed, we don't have variables/optimizers anymore

**Review by bani-intelaipg — APPROVED** [review]

File: ngraph_bridge/ngraph_utils.cc (hunk 2 above)
Comment (adk9) [reply ngraph_bridge/ngraph_utils.cc:465]: Removed.

File: ngraph_bridge/ops/ngraph_ops.cc (hunk 1 above)
Comment (adk9) [reply ngraph_bridge/ops/ngraph_ops.cc:1]: Ok, I'll do the refactoring to use canonical TF convention in a follow-up PR.

Comment (adk9) [reply ngraph_bridge/ops/ngraph_ops.cc:1]: See #710

**Pull request merged**